composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/booking/BookingScreen.kt

```diff
@@ -439,79 +442,88 @@ private fun BookingScreenContent(
             )
         }
 
-        Dialog(
-            content = {
-                BookingRepeatCard(
-                    onSelected = onClickOpenBookRepeat,
-                    modifier = Modifier.padding(horizontal = 16.dp).align(Alignment.Center),
-                    weekDays = if (bookingPeriod is BookingPeriod.Week)
-                        bookingPeriod.selectedDayOfWeek
-                    else emptyList()
-                )
-            },
-            onDismissRequest = onClickCloseRepeatDialog,
-            showDialog = showRepeatDialog,
-            modifier = Modifier.padding(horizontal = 16.dp).align(Alignment.Center)
-        )
+        if (showRepeatDialog) {
```

Comment:
maybe better use when

Reply:
`when` will not work because you need to show one `dialog` on top of the other